### ISSUE TITLE
tensorflow and keras version

### ISSUE THREAD

**Issue by dinglei0719** [masyagin1998/robin#14]
Hi,

thanks a lot for your work.
would you please share the tensorlow and keras version used in this work?
it seems like that it cannot work with the latest version.

thank you

**Comment by masyagin1998** [comment]
Hi!

You can find them in `requirements.txt`.

**Comment by lzltommy0805** [comment]
> Hi!
> 
> You can find them in `requirements.txt`.

Hi! What is the version of tensorflow? thx!

**Comment by lukeabela38** [comment]
@dinglei0719 I went ahead and worked with the project - use tensorflow 2.9.1 and the contents of requirements.txt:

absl-py==1.0.0
alt-model-checkpoint==1.0.1
astunparse==1.6.3
Augmentor==0.2.3
cachetools==5.1.0
certifi==2022.5.18.1
charset-normalizer==2.0.12
flatbuffers==1.12
future==0.18.3
gast==0.4.0
google-auth==2.6.6
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.46.3
h5py==3.6.0
idna==3.3
imageio==2.4.1
importlib-metadata==4.11.4
keras==2.9.0
Keras-Preprocessing==1.1.2
libclang==14.0.1
Markdown==3.3.7
numpy==1.22.0
oauthlib==3.2.0
opencv-python==4.7.0.68
opt-einsum==3.3.0
packaging==21.3
Pillow==9.4.0
protobuf==3.19.4
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyparsing==3.0.9
requests==2.27.1
requests-oauthlib==1.3.1
rsa==4.8
six==1.16.0
tensorboard==2.9.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow-cpu==2.9.1
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
termcolor==1.1.0
tqdm==4.64.1
typing_extensions==4.2.0
urllib3==1.26.9
Werkzeug==2.1.2
wrapt==1.14.1
zipp==3.8.0
